### PR TITLE
Benchmark for un-triaged image results on Flutter Gold

### PR DESCRIPTION
**Pull request by Piinks**
## Description

This adds a simple benchmark that tracks the number of un-triaged digests on Flutter Gold.
Having this benchmark in place will help safe guard against forgotten image changes as we move way from using flutter/goldens and towards Flutter Gold. :)
Related work:
- **Bot support:** https://github.com/flutter/cocoon/pull/409 
- **Local & Pre-submit support:** ~https://github.com/flutter/flutter/pull/39853~ https://github.com/flutter/flutter/pull/40710
- **Design doc:** [flutter.dev/go/golden-workflow](https://docs.google.com/document/d/1MuIUz9pyE_bBZPbtMCj3pYgkdgG4s4Egh6FMknTngKw/edit?usp=sharing) 

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: dev/devicelab/bin/tasks/flutter_gold_triage.dart [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const String _kTriageCountKey = 'count';
+
+Future<TaskResult> getTriageCount() async {
+  final HttpClient skiaClient = HttpClient();
+  int digestCount = 0;
+  try {
+    await skiaClient.getUrl(Uri.parse(
+      'https://flutter-gold.skia.org/json/byblame?query=source_type%3Dflutter'
+    ))
+      .then((HttpClientRequest request) => request.close())
```
Comment (goderbauer) [inline dev/devicelab/bin/tasks/flutter_gold_triage.dart]: Can you use await instead of the then closures?

Comment (Piinks) [reply dev/devicelab/bin/tasks/flutter_gold_triage.dart]: Done!

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by Piinks** [comment]
Hold on merging. Checking with @kjlubick on sanity of this API call. 🙂

File: dev/devicelab/bin/tasks/flutter_gold_triage.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import 'package:flutter_devicelab/framework/framework.dart';
+
+const String _kTriageCountKey = 'count';
+
+Future<TaskResult> getTriageCount() async {
+  final HttpClient skiaClient = HttpClient();
+  int digestCount = 0;
+  try {
+    final HttpClientRequest request = await skiaClient.getUrl(Uri.parse(
+      'https://flutter-gold.skia.org/json/byblame?query=source_type%3Dflutter'
```
Comment (kjlubick) [inline dev/devicelab/bin/tasks/flutter_gold_triage.dart]: https://flutter-gold.skia.org/json/trstatus is more direct endpoint here.

**Review by kjlubick — CHANGES_REQUESTED** [review]

**Review by kjlubick — APPROVED** [review]

**Pull request merged**